olamilekan000/Politico
# Ch #163564397 api create a political party

#### What does this PR do?

- Create a Political party with an API

#### Description of Task to be completed?

- Create a route or an endpoint to handle the request(POST)

- Handle all necessary errors

- Display necessary success messages

- Handle unauthorized requests to create a political party

- Test the route (Postman and Mocha)


#### How should this be manually tested?

- Postman should be used to test manually
- Pull the application locally: `git fetch origin ch-api-create-a-political-party-163564397`
- Install dependencies: `npm i`
- Start the dev server" `npm start`
`
localhost:8089:/api/v1/create_party 
POST {
  name: 'Young Shall Conquer Congress',
  partyEmail: 'youngonesshallconquer@yougins.we',
  hqAddress: 'Abuja',
  logoUrl: 'jdkjkeluheuikhffefef.jpg',
  partyDescr: 'A party for the youths... The young shall grow!',
  partyLeadersName: 'Yaro Bala',
}

`

#### What are the relevant pivotal tracker stories?
- Create a mockup data from political parties

- Create a route or an endpoint to handle the request(POST)

- Handle all necessary errors

- Display necessary success messages

- Handle unauthorized requests to create a political party

- Test the route (Postman and Mocha)

https://www.pivotaltracker.com/story/show/163564397
